config/production.js

```diff
@@ -4,7 +4,9 @@ const { consoleLogger } = require('../src/logger')
 module.exports = {
   appName: 'icarus-poc-backend-service',
   server: {
-    corsEnabledFor: ['*'],
+    corsEnabledFor: ['https://adalite.io', 'https://*.adalite.io', 'http://*.adalite.io',
```

Comment:
there should also be "https://localhost:*" since we started using it recently for development because of Ledger Nano support